_includes/tutorials/aggregating-minmax/ksql/code/src/statements.sql

```diff
@@ -0,0 +1,11 @@
+CREATE STREAM MOVIE_SALES (title VARCHAR, release_year INT, total_sales INT)
+             WITH (KAFKA_TOPIC='movie-ticket-sales',
+                   PARTITIONS=1,
+                   VALUE_FORMAT='avro');
+
+CREATE TABLE MOVIE_FIGURES_BY_YEAR AS
+    SELECT RELEASE_YEAR,
+        MIN(TOTAL_SALES) AS MIN__TOTAL_SALES,
```

Comment:
Why the double underscores?

Reply:
Robin is an old Oracle DBA. I've learned not to ask. (Seriously, since he's on PTO and we're talking about this, this is probably idiomatic among some set of database professionals. I don't mind the look, since it sets off the name of the aggregation from the column name.)

Reply:
Ahh, that idiom actually makes sense now that you point it out. I just didn't see the pattern at first :)